### PR TITLE
create SteadyTimeStepping and UnsteadyTimeStepping (4th PR)

### PR DESCRIPTION
**Pull request by feilin-flexcompute**
* create SteadyTimeStepping and UnsteadyTimeStepping classes

### CODE REVIEW

File: examples/run_case_from_inputs.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
         ),
         freestream=fl.FreestreamFromVelocity(velocity=286, alpha=3.06),
         fluid_properties=fl.air,
-        time_stepping=fl.TimeStepping(max_pseudo_steps=500),
+        time_stepping=fl.UnsteadyTimeStepping(max_pseudo_steps=500),
```
Comment (maciej-flexcompute) [inline examples/run_case_from_inputs.py:19]: I think this was meant to be Steady

File: flow360/component/flow360_params/flow360_params.py [hunk 2]
```diff
@@ -621,6 +618,29 @@ class Config(Flow360BaseModel.Config):
         deprecated_aliases = [DeprecatedAlias(name="physical_steps", deprecated="maxPhysicalSteps")]
 
 
+# pylint: disable=E0213
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:621]: do we need to disable pylint here?

File: flow360/component/flow360_params/flow360_params.py [hunk 3]
```diff
@@ -621,6 +618,29 @@ class Config(Flow360BaseModel.Config):
...
+class SteadyTimeStepping(BaseTimeStepping):
+    """
+    Steady time stepping component
+    """
+
+    physical_steps: Literal[1] = pd.Field(1, alias="physicalSteps", const=True)
+    time_step_size: Literal["inf"] = pd.Field("inf", alias="timeStepSize", const=True)
+
+
+# pylint: disable=E0213
```
Comment (maciej-flexcompute) [inline flow360/component/flow360_params/flow360_params.py:631]: do we need to disable pylint here?

File: flow360/component/flow360_params/flow360_params.py (hunk 2 above)
Comment (feilin-flexcompute) [reply flow360/component/flow360_params/flow360_params.py]: removed.

File: examples/run_case_from_inputs.py (hunk 1 above)
Comment (feilin-flexcompute) [reply examples/run_case_from_inputs.py]: fixed.

File: flow360/component/flow360_params/flow360_params.py (hunk 3 above)
Comment (feilin-flexcompute) [reply flow360/component/flow360_params/flow360_params.py]: removed.

**Review by maciej-flexcompute — APPROVED** [review]

**Pull request merged**